src/PointwiseFunctions/GeneralRelativity/Ricci.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines functions to calculate the Ricci tensor
+
+#pragma once
+
+#include "DataStructures/Tensor/IndexType.hpp"
+#include "DataStructures/Tensor/Symmetry.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes Ricci tensor from the (spatial or spacetime)
+ * Christoffel symbol of the second kind and its derivative.
+ *
+ * \details Computes Ricci tensor \f$\R_{ab}\f$ as:
+ * \f$ \R_{ab} = 1/2 ( \partial_c \Gamma^{c}_{ab} - \partial_{(b}
+ * \Gamma^{c}_{a)c}
+ * + \Gamma^{d}_{ab}\Gamma^{c}_{cd} - \Gamma^{d}_{ac} \Gamma^{c}_{bd}  ) \f$
+ * where \f$\Gamma^{a}_{bc} is the Christoffel symbol of the second kind.\f$
```

Comment:
I haven't tried rendering this, but it looks like the `\f$` is in the wrong place.

Reply:
Is there an easy way to generate the doxygen page so I can check myself?

Reply:
`make doc`

Reply:
.. and when you make doc, the doxygen will appear in docs/html of your build directory.  If you are using the docker container, you probably need to copy docs/html to your src directory so that you can view it outside the container.

Reply:
Thanks! I'll double-check this and fix this documentation if it's wrong as @wthrowe suspects.

Reply:
Docs are now fixed. Will was right that my TeX was incorrect. I verified that it now renders correctly.